staff/maider-hernandorena/spotify-app/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```

Comment:
You need to put a name into title tag. No more 'Document', please. It hurts me...

Reply:
Changed! Sorry, i didn't want to hurt you ... :D